### ISSUE TITLE
Hunger Increase Needs Tuning (way up or way down)

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLifeData7#1121]
Issue: The hunger increase is ineffective

Solution 1: Turn the rate down so that it is possible for families to split off and ESCAPE eves that keep returning to old bases

Solution 2: Turn it way up so that families actually die and cannot simply remain in an old, looted base for days/weeks

**Comment by SoloAceMouse** [comment]
> The hunger increase is ineffective

Ineffective at what exactly?
Preventing people from moving freely around the map.
If that is a problem, then free travel around the map would be restricted.

You present this as an issue because you do not like the way others play and enjoy the game. Framing your opinion on how the game should be played as issue with the game's mechanics is indicative of your agenda.

While we are all free to express our opinions, I will state I strongly disagree with yours.

-----

Short version:
This is a complaint which is being pushed as content issue and requires no action except to close this issue on Github.

**Comment by Forman9** [comment]
> Framing your opinion on how the game should be played as issue with the game's mechanics is indicative of your agenda.
This is where I disagree. Returning to old towns is closer to an exploit than an intentional mechanic.
Since this exploit is effectively blocking other players from enjoying at VERY LEAST 50% of game content, that makes it an ISSUE.

Sure people on github disagree.... maybe 5 people and that's mainly because most players and forum users do not have git accounts.

If you took a survey I would bet as many as 70-80% would agree this EXPLOIT needs to be PLUGGED.
A very basic indicator is the number of SID babies which clearly demonstrates players are constantly being (deliberately) funneled into game instantiations they do not want to play in.

I will go on to elaborate on the DELIBERATE nature of experienced players manipulating and exploiting design flaws in game mechanics to FORCE other players into THEIR instantiations. That is the very definition of a game exploit and an example of poor design, hence, an ISSUE.

**Comment by SoloAceMouse** [comment]
> If you took a survey I would bet as many as 70-80% would agree this EXPLOIT needs to be PLUGGED.

You are pulling these numbers out of your ass, and you know it.

I am talking with OHOL players every single day, and play 10-20 lives per week. Checking your leaderboard you play barely a fraction as much as I do and this has been true for over a year.

Over the past year, I've spoken with hundreds of players on the forums, reddit, github, and the Discord [where I primarily interact with other players out of game]. During this time, there have been differences of opinions on the validity of resettling. Some players are more or less in favor of it, especially regarding different circumstances.

You are the sole person I've interacted with who considers this an "exploit" however.

You are one guy ranting on Github, while literally hundreds of others enjoy the game fine in spite of your frustration.

Again, I'm sorry the game stopped being fun for you, but trying to get Jason to permanently nerf high-tech OHOL so you can play prim-tech is not representative of the community as a whole.

-----

tl;dr - This is an agenda-post masquerading as an issue/exploit. I understand we can't all be happy 100% of the time, but the proposed changes would disproportionately affect the community negatively.

**Comment by Arcurus** [comment]
There seems currently two issues:
1. Building up a new town is quite of impossible without being Eve since food is nerved with higher generations its difficult to make new towns in the wilderness and even worse you have no water source. I think food itself here is not a problem since you could bring it along water for sure is. This could maybe be solved if a new well is allowed if far far away from an existing well.
3. You can easily bring stuff from old Towns to new Eve towns which limit the new Town experience even more. This maybe could be solved in making travel on not roads lot harder with horse carts / trucks.

**Comment by Shadie116** [comment]
> Issue: The hunger increase is ineffective
> 
> Solution 1: Turn the rate down so that it is possible for families to split off and ESCAPE eves that keep returning to old bases
> 
> Solution 2: Turn it way up so that families actually die and cannot simply remain in an old, looted base for days/weeks

I agree, I think that the quantum theory is a theory and it's pretty annoying having people resettle when you're outside donkey town. Ugly old unartistic town made of plaster walls

**Comment by TwistedHBG** [comment]
Is today a bad github issue convention or something? Why didn't anyone inform me I have a bunch of stuff to share

**Comment by Shadie116** [comment]
Jeez @SoloAceMouse chill out, I get you take this game serious but no need to start using the big guns

**Comment by Tarr-OHOL** [comment]
The current iteration was made to encourage people to make different foods which has been overall successful. Towns now make a variety of food because it’s useful rather than in the past where mutton pies were dominating the meta.

Even with new people having decreased hunger it can be difficult to teach when people or yourself are starving. What we all find easy is near impossible for others.

**Comment by Shadie116** [comment]
I think we just need more food, how about apples. Wait no nvm they don't got apples in Europe. All they got is gooseberry bushes XD

**Comment by Forman9** [comment]
> Again, I'm sorry the game stopped being fun for you, but trying to get Jason to permanently nerf high-tech OHOL so you can play prim-tech is not representative of the community as a whole.

This is not at all the issue. I don't want to 'play prim tech' or NOT play advanced tech. What you seem to not understand is that a family can easily go from eve to the very end of the tech tree in ONE 24 hour period. It's the DAYS and DAYS spent AFTER the tech is in it's final stage that is annoying. 

> I talk with OHOL players every single day, and play 10-20 lives per week. Checking your leaderboard you play barely a fraction as much as I do and this has been true for over a year. As well, your interaction within the surrounding community is limited, with few modern players even knowing who you are.

I recently purchased a steam version but mostly played the regular pc version so can't see my total hours.
Based on how much I played during covid and in comparison with other games that are tracked, I wouldn't be surprised if my total was near 2000 hours. I played 8-10 hours nearly every day for months, paved probably over 200k tiles and left many colossal projects around with my name on them.  

> You are the sole person I've interacted with who considers this an "exploit" however.
In addition, I'm advanced/educated programmer with over 10 years of experience.
Game design is not my focus but the manner in which users are manipulating holes in the game design is certainly an exploit, not a feature.

> **Having free roam in an infinite map is not an "exploit", it's literally a key selling-point of the game for many players!**
Free roam is not the issue, exploiting game mechanics in order to force players into essentially their saved game, as apposed to allowing the game to progress natural is the problem. It takes considerable, deliberate effort for towns to last so long. Not families mind you, I mean towns.

> The claim that towns are lasting for weeks is a wholesale fabrication. 
I don't know whats been going on lately, but every time I log in every family has been on kerosene well. If you think distance is such a limiting factor you obviously don't recall the time families were separated by 100k and yet continued to resettle the same old towns. I don't have any hard numbers, but I'm sure those bases went on for 2 or more weeks.

> tl;dr - This is an agenda-post masquerading as an issue/exploit. I understand we can't all be happy 100% of the time, but the proposed changes would disproportionately affect the community negatively. 

You are simply incorrect. Game design, like any system design, is objective, not subjective. There is a logical, efficient way to design a game and there are infinite inefficient, illogical ways. You can certainly use a screwdriver as a hammer but that does not in fact make it a hammer and it is inefficient/illogical to do so.

**Comment by Shadie116** [comment]
Good point, I also think that bobbing for apples is sus. And yes game design is not subjective, a game isn't a good game if it fits the narrative of a single person with power. Democracy is an example

**Comment by SoloAceMouse** [comment]
> Even with new people having decreased hunger it can be difficult to teach when people or yourself are starving. What we all find easy is near impossible for others.

Yeah this is the point I wish to always emphasize.

We shouldn't make the game needlessly difficult for new players because 1000+ hour players are bored.

Balancing around this whining fool would make the game significantly harder for the people who already struggle.

**Comment by SoloAceMouse** [comment]
@Forman9 

You post this nonsense constantly, despite not playing the game.

When you post these garbage opinions on the forums, people rightfully shout them down as game-breakingly bad.
They do it on the github, as well.
They would do it on the discord and reddit, but you don't post them there.

There is a reason people disagree with these proposed changes.

It's because they're bad ideas.

**Comment by Shadie116** [comment]
> @Forman9
> 
> You post this stuff constantly, despite not playing the game.
> 
> When you post these opinions on the forums, people rightfully shout them down as game-breakingly bad. They do it on the github, as well, as I am doing currently. They would do it on the discord and reddit, but you don't post them there.
> 
> There is a reason people disagree with these proposed changes.
> 
> It's because they're bad ideas.


I don't know what to say but here's a death threat to calm the situation 💕💕
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/e7e6e2d8-1e63-44b7-adce-48f498536342)

**Comment by ShamWish** [comment]
> What you seem to not understand is that a family can easily go from eve to the very end of the tech tree in ONE 24 hour period. It's the DAYS and DAYS spent AFTER the tech is in it's final stage that is annoying.

The problem is the tech tree is so short not t

**Comment by Shadie116** [comment]
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/3cc1c4f4-1f7a-4b3d-913a-3e75cab378f8)

**Comment by Shadie116** [comment]
> @Forman9
> 
> You post this stuff constantly, despite not playing the game.
> 
> When you post these opinions on the forums, people rightfully shout them down as game-breakingly bad. They do it on the github, as well, as I am doing currently. They would do it on the discord and reddit, but you don't post them there.
> 
> There is a reason people disagree with these proposed changes.
> 
> It's because they're bad ideas.

"And don’t let us yield to temptation, but rescue us from the evil, gods kingdom, power, and glory, forever and ever, amen.

**Comment by Shadie116** [comment]
For the person named soloace mouse I will be processing a fuck to give... The message should send soon

**Comment by Shadie116** [comment]
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/1dfe7a28-28fd-479f-9ce9-af1192137ead)
Holy shit what the hell happened. That was my last fuck

**Comment by SoloAceMouse** [comment]
![shady bad](https://github.com/jasonrohrer/OneLifeData7/assets/147592460/72023909-4b3e-4b82-adb1-69cb2290677d)

**Comment by ShadyAbusesComplexWebappsOneFatalFlaw** [comment]
> ![shady bad](https://user-images.githubusercontent.com/147592460/279584341-72023909-4b3e-4b82-adb1-69cb2290677d.png)

White issue

**Comment by ShadyAbusesComplexWebappsOneFatalFlaw** [comment]
> ![shady bad](https://user-images.githubusercontent.com/147592460/279584341-72023909-4b3e-4b82-adb1-69cb2290677d.png)

Thanks for the fanart, I am very appreciative

**Comment by ShadyAbusesComplexWebappsOneFatalFlaw** [comment]
> ![shady bad](https://user-images.githubusercontent.com/147592460/279584341-72023909-4b3e-4b82-adb1-69cb2290677d.png)

![image](https://github.com/jasonrohrer/OneLifeData7/assets/149678753/7a529f8b-02ea-49fe-87e9-7da1099b7d0e)
